src/components/Text/src/Text.vue

```diff
@@ -249,6 +268,30 @@ export default {
 	--lh-7: calc(var(--lh-6) * var(--line-height-scale));
 }
 
+/* Variant defaults */
+.text_body,
+.text_label {
+	font-weight: var(--fontWeights-body, 400);
+	font-family: var(--fonts-body, sans-serif);
```

Comment:
why is the fallback `sans-serif` instead of `inherit`? please make it the latter